pay/contracts/dropper/HectorDropperFactory.sol

```diff
@@ -131,6 +131,14 @@ contract HectorDropperFactory is IHectorDropperFactory, Ownable {
         emit HectorDropperCreated(_token, hectorDropperContract);
     }
 
+    function releaseAirdrop(
+        address dropperContract,
+        address from,
+        uint256 index
+    ) external {
+        IHectorDropper(dropperContract).releaseAirdrop(from, index);
```

Comment:
@bejaxer what I am looking for is to iterate through all dropper contracts and release airdrop for this user with this index. Something like this:

function releaseSingleAirdrop(
        address[] calldata dropperContracts,
        address from,
        uint256 index
    ) external {
        uint256 length = dropperContracts.length;

        if (length != froms.length || length != indexes.length)
            revert INVALID_LENGTH();

        for (uint256 i = 0; i < length; i++) {
            IHectorDropper(dropperContracts[i]).releaseAirdrops(
                from,
                index
            );
        }
    }